sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_authentication.py

```diff
@@ -47,7 +47,7 @@ def authentication_with_api_key_credential_form_recognizer_client(self):
 
         form_recognizer_client = FormRecognizerClient(endpoint, AzureKeyCredential(key))
         # [END create_fr_client_with_key]
-        poller = form_recognizer_client.begin_recognize_receipts_from_url(self.url)
+        poller = form_recognizer_client.begin_recognize_content_from_url(self.url)
         receipt = poller.result()
```

Comment:
Should `receipt` be updated to `form`? (just for clarity) Same question for the change below.

Reply:
good point, going to change to `result` to match async sample